js/foundation.util.triggers.js

```diff
@@ -17,70 +18,120 @@ const triggers = (el, type) => {
     $(`#${id}`)[ type === 'close' ? 'trigger' : 'triggerHandler'](`${type}.zf.trigger`, [el]);
   });
 };
+
+var Triggers = {
```

Comment:
This is a longer-term thing, but moving to `let` and `const` would probably make some of this code more predictable. The eslint rules `pefer-const` and `no-var` can help with this, by identifying uses of `var`, or uses of `let` where the value is never reassigned.